apps/ledger-live-desktop/src/main/setup.js

```diff
@@ -23,7 +23,11 @@ if (process.env.DEV_TOOLS || process.env.DEBUG_LOGS) {
 }
 
 if (process.env.DESKTOP_LOGS_FILE) {
-  enableFileLogger();
+  let possiblydir = process.env.DESKTOP_LOGS_FILE;
```

Comment:
Not a case where we need to use `getEnv`?

Reply:
it doesn't feels necessary on this case because:
- it's only needed for LLD
- it's only needed for main thread